.drone.yml

```diff
@@ -166,9 +213,22 @@ services:
 matrix:
   include:
     # owncloud-coding-standard
-    - PHP_VERSION: 7.2
+    - PHP_VERSION: 5.6
       TEST_SUITE: owncloud-coding-standard
 
+    # phan
+    - PHP_VERSION: 7.2
+      OC_VERSION: daily-master-qa
+      TEST_SUITE: phan
+      DB_TYPE: sqlite
+      NEED_CORE: true
+
+    - PHP_VERSION: 7.0
+      OC_VERSION: daily-stable10-qa
+      TEST_SUITE: phan-70
```

Comment:
phan on PHP7.0 needs to be run with core ``stable10``
PHP7.0 is no longer supported with core ``master``. When I tried that, I got ``phan`` errors reported in drone https://drone.owncloud.com/owncloud/testing/235/29
```
php -d zend.enable_gc=0 vendor-bin/phan/vendor/bin/phan --config-file .phan/config.php --require-config-exists
lib/BigFileID.php:56 PhanUndeclaredClassMethod Call to method getDropSequenceSQL from undeclared class \Doctrine\DBAL\Platforms\AbstractPlatform
lib/BigFileID.php:57 PhanUndeclaredClassMethod Call to method getCreateSequenceSQL from undeclared class \Doctrine\DBAL\Platforms\AbstractPlatform
make: *** [test-php-phan] Error 1
Makefile:80: recipe for target 'test-php-phan' failed
```

Those errors are not reported when PHP7.0 is combined with core ``stable10``

Reply:
@patrickjahns this is an example where "something went wrong" if we run ``phan`` + PHP 7.0 + core ``master``. So maybe our "standard" way to run ``phan`` should have a matrix entry for ``phan`` + PHP 7.0 + core ``stable10`` an another matrix entry for ``phan`` + PHP 7.1/7.2 + core ``master`` ?